core/storage/user/gae_models.py

```diff
@@ -2565,9 +2566,9 @@ def get_model_association_to_user():
 
     @classmethod
     def get_export_policy(cls):
-        """Model does not need to exported as it temporarily holds user
-        requests for data deletion, and does not contain any information
-        relevant to the user for data export.
+        """Model contains data corresponding to a user, but model does not need
+        to exported as it temporarily holds user requests for data deletion, and
```

Comment:
Typo: ``...does not need to be exported...``

Reply:
Done